css/properties.json

```diff
@@ -1237,7 +1237,7 @@
     "status": "nonstandard"
   },
   "-webkit-mask-clip": {
-    "syntax": "[ border | border-box | padding | padding-box | content | content-box | text ]#",
+    "syntax": "<'mask-clip'>",
```

Comment:
`<-webkit-mask-clip>` is not the same as `<'mask-clip'>`. When I searched info about this property, I found something like this https://gist.github.com/webtobesocial/a27380c066a03e6c87b2#-webkit-mask-clip (and I remember a source about`*-box` values, can't found at the moment).